src/typesdoc.cpp

```diff
@@ -64,6 +64,22 @@ TypesDoc::TypesDoc()
     decimal_.init(Type::DECIMAL);
     decimal_.setDoc("Fixed precision decimal numbers, e.g. 0.34");
 
+    font_.init(Type::FONT);
+    font_.setDoc(
+         "A font specification (font family with modifiers regarding size, "
+         "weight, etc.) in one of the following formats:\n"
+         "\n"
+         "- Fontconfig description. This supports antialiased fonts,\n"
+         "  for example:\n"
+         "  * 'Dejavu Sans:pixelsize=12'\n"
+         "  * 'Bitstream Vera Sans:size=12:bold'\n"
+         "\n"
+         "- X logical font description (XLFD), as provided by the\n"
+         "  xfontsel tool. No antialiasing is supported, here but this\n"
```

Comment:
Shift the comma?
"No antialiasing is supported here, but..."

Reply:
Yes! it's fixed.

Reply:
Thanks!